docs/content-development/guide.md

```diff
@@ -559,14 +2041,14 @@ Let's go through the **Implementation** and try to understand, what is happening
 > ```
 > You can read more about policies on the [Policy overview](../feature/policies/overview.md) page.
 
+> In this example, the default backend storage is used. If you would like to use for instance AWS Secrets Manager, you can read how to do this [here](../feature/storage-backends/aws-secrets-manager.md#use-the-storage-backend). 
```

Comment:
Should we mention backend storage in some other places as well? I decided to do it in the place where it makes sense. Also, I didn't add `backend.context` because currently, we do not use it here. It probably will be updated after the implementation of Helm Storage.

Reply:
Sure, I wasn't sure if we want to demonstrate only the feature connected with our examples or all features connected with the content development. Now, it is clear.